examples/nodejs/cache/basic.ts

```diff
@@ -1,36 +1,133 @@
+  // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+  console.log(`Number as bytes: ${numAsBytes}`);
+
+  const storageClient = new PreviewStorageClient({
     configuration: Configurations.Laptop.v1(),
     credentialProvider: CredentialProvider.fromEnvironmentVariable({
       environmentVariableName: 'MOMENTO_API_KEY',
     }),
-    defaultTtlSeconds: 60,
   });
 
-  const createCacheResponse = await momento.createCache('cache');
-  if (createCacheResponse instanceof CreateCache.AlreadyExists) {
-    console.log('cache already exists');
-  } else if (createCacheResponse instanceof CreateCache.Error) {
-    throw createCacheResponse.innerException();
-  }
+  const storeName = 'store';
+  const putResponse = await storageClient.putInt(storeName, 'numberKey', 42);
+  // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+  console.log(`PutResponse: ${putResponse}`);
+  const getResponse = await storageClient.get(storeName, 'numberKey');
+  // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+  console.log(`GetResponse: ${getResponse}`);
+  const numberVal = getResponse.value();
+  // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+  console.log(`Number value: ${numberVal}`);
+  console.log(`Type of number value: ${typeof numberVal}`);
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  console.log(`YOLO happy path number value: ${getResponse.value()!.int()!}`);
 
-  console.log('Storing key=foo, value=FOO');
-  const setResponse = await momento.set('cache', 'foo', 'FOO');
-  if (setResponse instanceof CacheSet.Success) {
-    console.log('Key stored successfully!');
-  } else {
-    console.log(`Error setting key: ${setResponse.toString()}`);
-  }
+  const numberNumberVal: number = (function () {
+    console.log(`Get response type is: ${getResponse.type}`);
+    switch (getResponse.type) {
+      case StorageGetResponse.Success: {
+        console.log('get response is a StorageGetResponse.Success');
+        return getResponse.value()!.int()!;
+      }
+      case StorageGetResponse.Error: {
+        throw new Error('Not implemented yet: StorageGetResponse.Error case');
+      }
+    }
+  })();
+
+  console.log(`Number number value: ${numberNumberVal}`);
 
-  const getResponse = await momento.get('cache', 'foo');
-  if (getResponse instanceof CacheGet.Hit) {
-    console.log(`cache hit: ${getResponse.valueString()}`);
-  } else if (getResponse instanceof CacheGet.Miss) {
-    console.log('cache miss');
-  } else if (getResponse instanceof CacheGet.Error) {
-    console.log(`Error: ${getResponse.message()}`);
+  const notFoundGetResponse = await storageClient.get(storeName, 'notFoundKey');
+  // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+  console.log(`NotFoundGetResponse: ${notFoundGetResponse}`);
+  const notFoundNumber = notFoundGetResponse.value();
```

Comment:
and this is what i was trying to get us to in terms of being able to clearly identify a "not found"